docs/samples/customdomain/README.md

```diff
@@ -0,0 +1,134 @@
+# Setting up custom domain name
+
+## Setup
+
+1. Your ~/.kube/config should point to a cluster with [KFServing installed](https://github.com/kubeflow/kfserving/blob/master/docs/DEVELOPER_GUIDE.md#deploy-kfserving).
+2. Your cluster's Istio Ingress gateway must be network accessible.
+3. You have a custom domain configured to route incoming traffic to the cluster's Ingress gateway.
+
+## Create the Ingress resource
```

Comment:
Maybe add some background on why we need the Kubernetes ingress here. I know many Cloud providers have domain for their Kubernetes Ingress so we are doing this. Try to clarify your custom domain is routed to your Kubernetes Ingress, so you need this extra step.